src/base_media_base.h

```diff
@@ -170,8 +162,8 @@ struct BaseSet {
 template <class Tbase_set>
 class BaseMedia : FileScanner {
 protected:
-	static inline Tbase_set *available_sets = nullptr; ///< All available sets
-	static inline Tbase_set *duplicate_sets = nullptr; ///< All sets that aren't available, but needed for not downloading base sets when a newer version than the one on BaNaNaS is loaded.
+	static inline std::list<Tbase_set> available_sets; ///< All available sets
+	static inline std::list<Tbase_set> duplicate_sets; ///< All sets that aren't available, but needed for not downloading base sets when a newer version than the one on BaNaNaS is loaded.
```

Comment:
why std::list over std::vector?

Reply:
The original data structure is a list, and it appears pointer stability is required.